tr_pud-ud-test.conllu

```diff
@@ -3857,7 +3857,7 @@
 # text = Seçim bölgesi, seçmenlerin yüzde 62'si AB'den ayrılmayı destekleyen Kuzey Kesteven konsey bölgesindedir.
 # text_en = The constituency is in the council area of North Kesteven, where 62% of voters backed leaving the EU.
 1	Seçim	seçim	NOUN	NN	Number=Sing	2	nmod:poss	_	_
-2	bölgesi	bölge	NOUN	NN	Case=Nom|Number=Sing|Number[psor]=Sing|Person[psor]=3	13	nsubj	_	SpaceAfter=No
+2	bölgesi	bölge	NOUN	NN	Case=Nom|Number=Sing|Number[psor]=Sing|Person[psor]=3	13	nsubj:outer	_	SpaceAfter=No
```

Comment:
Why is this labeled as outer subject? Shouldn't the other subject, _seçmenlerin yüzde 62'si_, be rather attached to a nested clause?

Reply:
Hm, I took the definition word by word: _`nsubj:outer` specifies a nominal subject of a copular clause whose predicate is itself a clause_, and concluded that _Seçim bölgesi_ is the subject (but not nominal, in fact). Do you see it the other way round (_62'si_ is `nsubj:outer`)?

Reply:
_62'si_ here is the subject of the _destekleyen_ which is a participle.

On a general note: the annotations in this treebank is really in a bad shape. There has been some fixes from BOUN people a few years ago, but as far as I know they were not applied. I think we should re-activate that effort and apply their changes before diverging from the base a lot.

Reply:
I see, I wanted only to correct the things detected by the automatic validator to be sure that the PUD treebanks won't be excluded from the future versions, because I think the PUds are really useful from a typological point of view. But I agree a new effort may be necessary to revisit things ....

Reply:
I agree. It would be a pity if we drop PUD treebanks. I'd be happy to have a look at the Turkish one - at least to get it validate by next data freeze, but problems are beyond the issues of validation. I hope we can also improve the actual annotations. I will start contacting the BOUN people. Maybe we can factor their changes in before working on the validation issues.